src/components/registration/MasterFormMentee.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from 'react';
+import { connect } from 'react-redux';
+import * as actions from '../../redux/actions';
+import { withRouter } from 'react-router';
+import { Form, Tooltip, Input } from 'antd';
+import { Formik, useField, useFormikContext } from 'formik';
+import * as Yup from 'yup';
+//https://codesandbox.io/s/building-multi-step-form-with-formik-yup-3ro0d?file=/src/App.js
+const MasterFormMentee = () => {
+  const MyTextInput = ({ label, ...props }) => {
```

Comment:
these subcomponents can probably be lifted? unless I'm missing it, it doesn't look like they're using anything inside of the MasterFormMentee's scope.